word_language_model/README.md

```diff
@@ -4,11 +4,17 @@ This example trains a multi-layer RNN (Elman, GRU, or LSTM) on a language modeli
 By default, the training script uses the Wikitext-2 dataset, provided.
 The trained model can then be used by the generate script to generate new text.
 
-```bash
-python main.py --cuda --epochs 6        # Train a LSTM on Wikitext-2 with CUDA
-python main.py --cuda --epochs 6 --tied # Train a tied LSTM on Wikitext-2 with CUDA
-python main.py --cuda --tied            # Train a tied LSTM on Wikitext-2 with CUDA for 40 epochs
-python generate.py                      # Generate samples from the trained LSTM model.
+The recent updates support a Transformer model, which is composed of encoder and decoder with multihead attention layers.
```

Comment:
I wouldn't mention that there was an update, just that the Transformer model exists, since at some point the update won't be recent anymore and then this needs to be updated.